package.json

```diff
@@ -22,9 +22,11 @@
   },
   "main": "index.js",
   "dependencies": {
+    "@types/lodash": "^4.14.167",
```

Comment:
I think those types should go into the dev dependencies

Reply:
Done